src/components/svelte-web.ts

```diff
@@ -46,6 +46,10 @@ export default function registerWebComponent(
   component: any,
   { name, mode }: Options
 ) {
+  if (!customElements) {
```

Comment:
I think this will still error - we probably need to check globalThis.customElements